src/main/java/level5.java

```diff
@@ -0,0 +1,144 @@
+        done[i] = false;
+    }
+
+    public static void deadline(int i, String line, Task[] tasks, boolean[] done, String[] type) {
+        int slash = line.indexOf("/");
+        String description = line.substring(8, slash-1);
+        String by = line.substring(slash+4);
+        tasks[i] = new Deadline(description, by);
+        type[i] = "[D]";
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + type[i] + "[ ]" + tasks[i]);
+        System.out.println("Now you have " + (i + 1) + " tasks in the list.");
+        done[i] = false;
+    }
+
+    public static void event(int i, String line, Task[] tasks, boolean[] done, String[] type) {
+        int slash1 = line.indexOf("/");
+        int slash2 = line.indexOf("/", slash1+1);
+        String description = line.substring(5, slash1-1);
+        String from = line.substring(slash1+6, slash2-1);
+        String to = line.substring(slash2+4);
+        tasks[i] = new Event(description, from, to);
+        type[i] = "[E]";
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + type[i] + "[ ]" + tasks[i]);
+        System.out.println("Now you have " + (i + 1) + " tasks in the list.");
+        done[i] = false;
+    }
+    public static void main(String[] args){
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        String line;
+        Task[] tasks = new Task[100];
+        boolean[] done = new boolean[100];
+        String[] type = new String[100];
+        int i = 0;
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+            if (line.startsWith("bye")) {
+                break;
+            }
+            if (line.startsWith("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int j = 0; j < i; j++) {
+                    System.out.print(j+1);
+                    if (done[j]){
+                        System.out.print("." + type[j] + "[X]");
+                    }
+                    if (!done[j]){
+                        System.out.print("." + type[j] + "[ ]");
+                    }
+                    System.out.println(tasks[j]);
+                }
+            }
+            else if (line.startsWith("mark")) {
+                try {
+                    mark(i, line, tasks, done, type);
+                } catch (NumberFormatException | IndexOutOfBoundsException | DukeException e) {
+                    System.out.println("Oops! That isn't a task in your list.");
+                    System.out.println("Please input only one integer after 'mark', separated by a single whitespace.");
+                    System.out.println("There are " + i + " tasks in your list, so do not input an integer " +
+                            "smaller than 1 or larger than " + i + ".");
```

Comment:
```suggestion
                    System.out.println("There are " + i + " tasks in your list, so do not input an integer " 
                            + "smaller than 1 or larger than " + i + ".");
```
Try to keep the operators in front of the new line to prevent readers from having to refer to the previous line